### PR TITLE
Add .NET 9 (#204)

### LINKED ISSUE #204: Add support for .Net 9 / EF 9
Please add support for .Net 9 / EF 9

This [PR](205) should resolve this issue.

### PR DESCRIPTION
**Pull request by rizi**
Add support for .Net 9 / EF 9.

This should fix #204.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
...
 | LinqKit.Microsoft.EntityFrameworkCore 7 | [![Nuget](https://img.shields.io/badge/nuget-v7.1.5-blue) ![Nuget](https://img.shields.io/nuget/dt/LinqKit.Microsoft.EntityFrameworkCore)](https://www.nuget.org/packages/LinqKit.Microsoft.EntityFrameworkCore/7.0.0) | Microsoft.EntityFrameworkCore<ul><li>≥ 7.0.0</li></ul> | <ul><li>net6.0</li><li>net7.0</li></ul>|
 | LinqKit.Microsoft.EntityFrameworkCore 8 | [![Nuget](https://img.shields.io/badge/nuget-v8.1.5-blue) ![Nuget](https://img.shields.io/nuget/dt/LinqKit.Microsoft.EntityFrameworkCore)](https://www.nuget.org/packages/LinqKit.Microsoft.EntityFrameworkCore/8.0.0) | Microsoft.EntityFrameworkCore<ul><li>≥ 8.0.0</li></ul> | <ul><li>net8.0</li></ul>|
+| LinqKit.Microsoft.EntityFrameworkCore 9 | [![Nuget](https://img.shields.io/badge/nuget-v8.1.5-blue) ![Nuget](https://img.shields.io/nuget/dt/LinqKit.Microsoft.EntityFrameworkCore)](https://www.nuget.org/packages/LinqKit.Microsoft.EntityFrameworkCore/9.0.0) | Microsoft.EntityFrameworkCore<ul><li>≥ 9.0.0</li></ul> | <ul><li>net9.0</li></ul>|
```
Comment (StefH) [inline README.md:22]: badge version probably needs to be v9.1.5 here.

File: version.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <Project>
   <PropertyGroup>
-    <PatchVersion>5</PatchVersion>
+    <PatchVersion>6</PatchVersion>
```
Comment (StefH) [inline version.xml:3]: just keep 5

I'll do the version update in a separate commit

File: README.md (hunk 1 above)
Comment (rizi) [reply README.md:22]: Thx, for the input, I fixed it.

File: version.xml (hunk 2 above)
Comment (rizi) [reply version.xml:3]: Thx for the input, I fixed it.

**Review by StefH — APPROVED** [review]

**Pull request merged**